### DISCUSSION TITLE
HI How to insert data to multiple tables

### DISCUSSION THREAD

**Discussion by donPuerto** [supabase/supabase#2796] [Questions] [answered]
Hi Im trying to add ID to the related tables, since I have multiple tables, I need to insert/upsert into multiple tables. I don't  know if this is the right pattern. Let me know

ie
user has one to one relationship profile
user has one to many relationship to address

**Comment by steve-chavez** [comment, marked as answer]
Hey @donPuerto,

You can do so through a SQL function and call it on supabase-js through [rpc](https://supabase.io/docs/reference/javascript/rpc).
